python/cuml/common/numba_utils.py

```diff
@@ -37,11 +35,11 @@ def gpu_zeros_2d(out):
         out[i][j] = 0
 
 
-def zeros(size, dtype, order='F'):
+def zeros(size, dtype, order="F"):
```

Comment:
This function is deprecated and is only used in a couple of non cumlarray using files, @jakirkham any chance you could open a gh issue to remove those uses to be able to remove this `numba_utils` file that is no longer necessary?